src/tools/gophish_complete.py

```diff
@@ -3,14 +3,15 @@
 """Complete a campaign in GoPhish and/or output a GoPhish campaign summary.
 
 Usage:
-  gophish-complete (--auto=CAMPAIGN_ID | --complete | --summary ) [--log-level=LEVEL] SERVER API_KEY
+  gophish-complete (--complete | --summary) [--log-level=LEVEL] SERVER API_KEY
+  gophish-complete CAMPAIGN_NAME (--complete | --summary) [--log-level=LEVEL] SERVER API_KEY
```

Comment:
Can't we consolidate these two docopt lines into a single line like this?
```
gophish-complete (--complete | --summary) [--campaign=NAME] [--log-level=LEVEL] SERVER API_KEY
```

Reply:
Great idea! Implemented this change in commit fa471ad. 

Successfully tested with both `-C NAME` and `--Campaign=NAME` to ensure the desired campaign was used.  Also verified running without the `--Campaign` option did not impact the manual process.